WordPress/src/main/res/values/strings.xml

```diff
@@ -558,6 +558,8 @@
     <string name="older_last_week">Older than a week</string>
     <string name="older_month">Older than a month</string>
     <string name="error_notification_open">Could not open notification</string>
+    <string name="ignore">Ignore</string>
+    <string name="push_auth_expired">The login request has expired. Log in to WordPress.com to try again.</string>
```

Comment:
We use different wordage for logging in ("login" and "log in" and "sign in") in different places in the app. The most prevalent is "sign in" - can we use that here, too?